canonical/canonical.com
# Update URL path for /ai/infrastructure

## Done

- Update URL from /solutions/ai-infrastructure to /solutions/ai/infrastructure

## QA

- Go to https://canonical-com-1307.demos.haus/solutions/ai/infrastructure
- See that the page loads as expected
- Go to https://canonical-com-1307.demos.haus/solutions/ai/infrastructure#get-in-touch
- See that form modal pops up
- Go to https://canonical-com-1307.demos.haus/solutions/ai/infrastructure#contact-form-success
- See that the index page has success form at the top of the page

## Issue / Card

Fixes [WD-13436](https://warthogs.atlassian.net/browse/WD-13436)

## Screenshots

[if relevant, include a screenshot]


[WD-13436]: https://warthogs.atlassian.net/browse/WD-13436?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
